pgxn/neon/libpagestore.c

```diff
@@ -1321,6 +1326,16 @@ pg_init_libpagestore(void)
 							PGC_USERSET,
 							0,	/* no flags required */
 							NULL, (GucIntAssignHook) &readahead_buffer_resize, NULL);
+	DefineCustomIntVariable("neon.readahead_getpage_pull_timeout",
+							"readahead response pull timeout",
+							"Time between active tries to pull data from the "
+							"PageStream connection when we have pages which "
+							"were read ahead but not yet received.",
+							&readahead_getpage_pull_timeout_ms,
+							-1, -1, INT_MAX,
```

Comment:
```suggestion
							0, 0, 5 * 60 * 1000,
```

-1 and 0 would both disable the feature. Let's only accept 0. Please also mention that in a comment somewhere

Also have a more reasonable maximum. I'm not sure if INT_MAX would actually work or overflow some math somewhere, but instead of trying to figure that out, make 5 minutes the maximum. That seems plenty.